src/lang/swedish.lng

```diff
@@ -1,15 +1,15 @@
 ##grflangid 0x2e
 
 STR_GRF_NAME                        :{TITLE} {VERSION}
-STR_GRF_DESC                        :{ORANGE}Vanilla-inspired industries which require serving and growing towns. {BLACK}Recommended for use with a growth script like Renewed Village Growth.{}{}{ORANGE}Visit website for more info!{}{}{BLACK}Created by 2TallTyler.
+STR_GRF_DESC                        :{ORANGE}Vanilla-inspererade industrier some kräver betjäning och tillväxt i städer. {BLACK}Rekommenderas för användning med ett tillväxtskript som till exempel Renewed Village Growth. {}{}{ORANGE}Besök hemsidan för mer information!{}{}{BLACK}Skapad av 2TallTyler.
 
 STR_WEBSITE                         :{WEBSITE}
 
 STR_ERROR_GAME_VERSION              :{TITLE} kräver OpenTTD version 1.10.0, JGR version 0.34, eller högre.
 STR_ERROR_INCOMPATIBLE_SET          :{YELLOW}{TITLE}{WHITE} är inkompatibel med {YELLOW}{STRING}{WHITE}.
 
-STR_PARAM_PRIMARY_QUADRANTS_NAME    :Primary industry regions
-STR_PARAM_PRIMARY_QUADRANTS_DESC    :{LTBLUE}Default value: {ORANGE}On{}{WHITE}If enabled, primary industries spawn only in certain parts of the map.
+STR_PARAM_PRIMARY_QUADRANTS_NAME    :Primära industriområden
+STR_PARAM_PRIMARY_QUADRANTS_DESC    :{LTBLUE}Förvalt värde: {ORANGE}På{}{WHITE}Om aktiverad, vissa industrier dyker endast upp i vissa delar av kartan.
```

Comment:
Never use applications in Swedish, so this might not be what's most commonly used. But should make sense at least.